recipes/recipes_emscripten/lfortran/recipe.yaml

```diff
@@ -33,6 +33,13 @@ requirements:
   host:
   - zlib
 
+tests:
+  - script:
+    - node ${PREFIX}/bin/test_lfortran.js
+    requirements:
+      build:
+        - nodejs
```

Comment:
I think nodejs should be a **run** requirement not a build requirement of the tests

Reply:
aahh no..wait..it must be a build requirement since we are cross compiling...sorry....

sorry for the confusion and wasted time